cumulus/parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -41,7 +41,11 @@ use sp_std::prelude::*;
 use sp_version::NativeVersion;
 use sp_version::RuntimeVersion;
 
-use cumulus_primitives_core::{AggregateMessageOrigin, ParaId};
+use bridge_hub_common::{
+	message_queue::{NarrowOriginToSibling, ParaIdToSibling},
+	AggregateMessageOrigin,
+};
```

Comment:
Are we gonna merge our changes to all bridge-hub runtimes including Kusama and Polkadot, or just focus on the Rococo runtime?

Reply:
Hmmm that's a good point. All our other config changes are just on the Rococo runtime, so let me revert the Kusama, Westend and Polkadot changes.

Reply:
I get compiler errors if I don't update all the runtimes. I think its because it all the bridge hub runtime use `bridge_hub_common` for `NarrowOriginToSibling` and `ParaIdToSibling`, and then `AggregateMessageOrigin` needs to be consistent (can't mix using `AggregateMessageOrigin` in `bridge_hub_common` and then the runtime uses the `AggregateMessageOrigin` in the cumulus primitives.